libs/backend/src/cast_vote_records/export.ts

```diff
@@ -526,47 +559,45 @@ export async function updateCreationTimestampOfDirectoryAndChildrenFiles(
 
 /**
  * File creation timestamps could reveal the order in which ballots were cast. To maintain
- * voter privacy, every time a ballot is cast, we randomly select 1 or 2 previously added cast vote
- * records and update their creation timestamps to the present. This approach is equivalent to
- * moving random ballots to the top of a stack every time a new ballot is added to the stack, a
- * kind of shuffling as we go.
+ * voter privacy, every time a ballot is cast, we randomly select 1 or 2 previous ballots and
+ * update their creation timestamps to the present. This approach is equivalent to moving random
+ * ballots to the top of a stack every time a new ballot is added to the stack, a kind of shuffling
+ * as we go.
  *
- * This shuffling is irrelevant for full exports, where we already iterate over cast vote records
- * in an order independent of creation timestamp.
+ * This shuffling is irrelevant for full exports, where we already iterate over ballots in an order
+ * independent of creation timestamp.
  */
 async function randomlyUpdateCreationTimestamps(
   exportContext: ExportContext,
   exportDirectoryPathRelativeToUsbMountPoint: string,
-  options: { castVoteRecordIdToIgnore?: string } = {}
+  options: { subDirectoryNameToIgnore?: string } = {}
```

Comment:
Updated the variable names in this function because we're now shuffling not only cast vote record sub-directories but also sub-directories for rejected sheets, i.e. all export sub-directories